content/en/docs/tasks/tools/install-kubectl-linux.md

```diff
@@ -100,13 +100,35 @@ For example, to download version {{< param "fullversion" >}} on Linux, type:
 ### Install using native package management
 
 {{< tabs name="kubectl_install" >}}
-{{< tab name="Ubuntu, Debian or HypriotOS" codelang="bash" >}}
-sudo apt-get update && sudo apt-get install -y apt-transport-https gnupg2 curl
-curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key add -
-echo "deb https://apt.kubernetes.io/ kubernetes-xenial main" | sudo tee -a /etc/apt/sources.list.d/kubernetes.list
-sudo apt-get update
-sudo apt-get install -y kubectl
-{{< /tab >}}
+{{% tab name="Ubuntu, Debian or HypriotOS" %}}
```

Comment:
good opportunity to rename this to "Debian based"

we did a similar change elsewhere
/cc @sftim

Reply:
I'm also happy if that lands in a separate PR.